mesonbuild/backend/ninjabackend.py

```diff
@@ -1863,6 +1863,9 @@ def generate_single_compile(self, target, outfile, src, is_generated=False, head
         # near the end since these are supposed to override everything else.
         commands += self.escape_extra_args(compiler,
                                            target.get_extra_args(compiler.get_language()))
+        # Add per-target compiler standard version.
+        if(target.get_compiler_standard(compiler.get_language()) != None):
```

Comment:
__[flake8]__
* [E711] comparison to None should be 'if cond is not None:'

<!-- Comment by [SideCI](https://sideci.com) -->